docs/site/pages/content/dsl.mdx

```diff
@@ -94,15 +94,25 @@ Similarly, if a single asset is expected but a list of them is found instead, Pl
 
 ### Templates
 
-Templates are included via the `@player-tools/dsl` package. This can be used in any asset slot:
+Templates are included via the `@player-ui/dsl` package. This can be used in any asset slot:
```

Comment:
I think `@player-tools/dsl` is the correct package. https://github.com/player-ui/tools/blob/main/language/dsl/BUILD#L4

Reply:
It seems both export `Template`. However, only the the one exported by `@player-ui/dsl` is ok to receive a `MakeBindingRefable`:

<img width="1143" alt="Screenshot 2024-01-12 at 8 09 47 AM" src="https://github.com/player-ui/player/assets/26394217/f2c459bb-0f2a-49e7-9cb1-954f54e88df3">

<img width="1909" alt="Screenshot 2024-01-12 at 8 09 28 AM" src="https://github.com/player-ui/player/assets/26394217/9850d88e-da7d-47ae-a3f7-d76189a7afaf">